modules/shared/api/postsApi.api.ts

```diff
@@ -26,9 +26,12 @@ export const postsApi = new postClient.PostsApi(
     basePath: 'https://pickify-posts-be-dev.m3ntorship.net/api',
   } as Configuration,
   undefined,
-  postsApiAxiosInstance,
```

Comment:
this change I am not sure about. if it works now without this instance we can merge it and come back to it later

Reply:
trust me :joy: 
it's useless since we are sending the Authorization with `getPosts` function

Reply:
not really. we might need it to add some other interceptor or defaults. but let's merge it like this for now.

Reply:
we will add another interceptor for this function but with no Authorization, i will test this solution later and let you know